schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/util/CustomParameterizedBlockJUnit4Runner.java

```diff
@@ -0,0 +1,242 @@
+/**
```

Comment:
@satishd on the note of licenses i note the code base has a lot of glassfish dependencies which is GPL afaik,  as this project is ASL is it not maybe better to look at apache projects for equiv replacements.